.github/workflows/release.yml

```
@@ -0,0 +1,161 @@
+      uses: actions/download-artifact@v4
+      with:
+        path: artifacts
+
+    - name: Create checksums
+      run: |
+        cd artifacts
+        find . -type f -name "digests-api-*" -exec sha256sum {} \; > ../checksums.txt
+        mv ../checksums.txt .
+
+    - name: Create Release
+      uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)